opensearch-notebooks/gradlew

```diff
@@ -1,19 +1,8 @@
 #!/usr/bin/env sh
 
 #
-# Copyright 2015 the original author or authors.
```

Comment:
this is from apache license. could you keep the original license, and add our full license on top?

Reply:
yeah okay